spec/models/user_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject do
+    User.create do |user|
+      user.name = 'Abdimulhin Adan'
+      user.photo = 'https://images.unsplash.com/photo-1533167649158-6d508895b680?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1032&q=80'
+      user.bio = 'Am a kenyan'
+    end
+  end
+  it 'name should be exist' do
+    expect(subject.name).to eql('Abdimulhin Adan')
+  end
```

Comment:
- [ ] You did a good job working on this milestone! 💯 However, I would suggest that rather than testing what's the actual name, you assert if it's valid or not and consider both cases! Here, for example, you could change this test to:

```suggestion
context 'when name exists' do
  it 'should be valid' do
    expect(subject).to be_valid
  end
end

context "when name doesn't exist" do
  it 'should not be valid' do
    subject.name = ''
    expect(subject).not_to be_valid
  end
end
```

Please consider applying this change! 🤓 🔧